src/main/kotlin/com/openlattice/mechanic/upgrades/GrantPublicSchemaAccessToOrgs.kt

```diff
@@ -0,0 +1,95 @@
+package com.openlattice.mechanic.upgrades
+
+import com.google.common.base.Preconditions
+import com.openlattice.IdConstants
+import com.openlattice.ResourceConfigurationLoader
+import com.openlattice.assembler.AssemblerConfiguration
+import com.openlattice.assembler.AssemblerConnectionManager
+import com.openlattice.assembler.PostgresDatabases
+import com.openlattice.assembler.PostgresRoles
+import com.openlattice.authorization.AclKey
+import com.openlattice.authorization.PrincipalType
+import com.openlattice.authorization.SecurablePrincipal
+import com.openlattice.datastore.util.Util
+import com.openlattice.mechanic.Toolbox
+import com.openlattice.organizations.roles.SecurePrincipalsManager
+import com.openlattice.postgres.DataTables
+import com.openlattice.postgres.PostgresColumn
+import com.openlattice.postgres.PostgresTable
+import com.openlattice.postgres.ResultSetAdapters
+import com.openlattice.postgres.streams.BasePostgresIterable
+import com.openlattice.postgres.streams.StatementHolderSupplier
+import com.zaxxer.hikari.HikariConfig
+import com.zaxxer.hikari.HikariDataSource
+import java.util.*
+
+class GrantPublicSchemaAccessToOrgs(
+        private val toolbox: Toolbox,
+        private val securePrincipalsManager: SecurePrincipalsManager) : Upgrade {
+    private val acmConfig: AssemblerConfiguration
+    private val PUBLIC_SCHEMA = "public"
+
+    init {
+        val acmConfig = ResourceConfigurationLoader.loadConfigurationFromResource("assembler.yaml", AssemblerConfiguration::class.java)
```

Comment:
You can just use the configuration from the MechanicUpgradePod. See also . in `OrganizationDbUserSetup `